### ISSUE TITLE
Text Alignment issue inside Mathjax Latex Array

### ISSUE THREAD

**Issue by arjunsuresh** [wkhtmltopdf/wkhtmltopdf#5028]
**wkhtmltopdf version(s) affected**: All versions

**OS information**
Ubuntu and MAC (possibly all)

**Description**
array environment inside Mathjax is not respecting text alignment with the generated PDF whereas all browsers does this correctly. A generated PDF is [here.](https://drive.google.com/file/d/1pjDP8iLLet_WyuKCPsQ6QYA8X124NtvP/view?usp=sharing)

**How to reproduce**
With [test1.html](https://drive.google.com/file/d/1IbJJSEzISdMjGIr_KdV84b6wqm5GEz_4/view?usp=sharing) please do
```wkhtmltopdf --javascript-delay 1000 test1.html test2.pdf``` 
**Expected behavior**
[As shown in the pic](https://drive.google.com/file/d/149n42C-mvtxkiiXuDK8cQAhl7WfdBK80/view?usp=sharing)

**Comment by MohammedZabiulla** [comment]
Just put 
style="align-items:left;
inside p tag 
![Screenshot 2021-06-27 092834](https://user-images.githubusercontent.com/55593729/123532369-46b81700-d72a-11eb-88da-6f2063f7dacc.png)

is this issue you were having?

**Comment by arjunsuresh** [comment]
Doesnt work for me. Did that really work for you?

**Comment by MohammedZabiulla** [comment]
> Doesnt work for me. Did that really work for you?

Yeah ,it worked perfectly for me.!

**Comment by arjunsuresh** [comment]
In the generated PDF? Which OS?

**Comment by MohammedZabiulla** [comment]
> In the generated PDF? Which OS?
Kali Linux

**Comment by arjunsuresh** [comment]
Strange. That has no effect on Ubuntu. `wkhtmltopdf 0.12.6 (with patched qt)`

**Comment by shashankrustagi** [comment]
Sir, It is working fine with me as well,
I am using MacOS 11.4 version

**Comment by shashankrustagi** [comment]
![Screenshot 2021-06-28 at 2 17 07 AM](https://user-images.githubusercontent.com/26660782/123558968-f4710780-d7b6-11eb-98aa-285c270483d6.png)

Working fine with me sir,
May be it is Operating system issue.

**Comment by arjunsuresh** [comment]
Which OS? Yes, it works on Arch Linux but not on Ubuntu or Mac.

**Comment by shashankrustagi** [comment]
No sir, It is working on MacOs 11.4, i am using MacOs.

**Comment by arjunsuresh** [comment]
Please see the steps I had mentioned to reproduce the issue -- it wont show on browser.

**Comment by shashankrustagi** [comment]
![Screenshot 2021-06-28 at 3 20 31 AM](https://user-images.githubusercontent.com/26660782/123560450-d2c84e00-d7bf-11eb-8d15-84ca66d2507f.png)

I reproduced, and found that, it works on my machine pretty well sir.
Thanks

**Comment by arjunsuresh** [comment]
Actually that table is not fully rendered. You can try increasing the JavaScript delay. But as posted on other issue, this might be related to the screen resolution I guess.

**Comment by shashankrustagi** [comment]
OK sir, let me try and post.

**Comment by shashankrustagi** [comment]
![Screenshot 2021-06-28 at 3 57 17 AM](https://user-images.githubusercontent.com/26660782/123561269-f17d1380-d7c4-11eb-99c6-cba66850a0ca.png)

Yup sir, working fine now
I guess both of the issues are resolved now?

**Comment by arjunsuresh** [comment]
No. This is the issue. Instead of "left aligned" Table contents are coming "center aligned"

**Comment by shashankrustagi** [comment]
sir, ya i tried it with 2800 as delay still it is not getting left aligned. let me read documentation of MathJax

**Comment by shashankrustagi** [comment]
sir, any progress?

**Comment by shashankrustagi** [comment]
Sir, I think the issue is with wkhtmltopdf, mathjax syntax here is not helping, it is redering issue.

**Comment by arjunsuresh** [comment]
Yes. It is. That's why issue is filed under wkhtmltopdf 🙂

**Comment by shashankrustagi** [comment]
yes sir.
